src/engraving/libmscore/staff.cpp

```diff
@@ -816,6 +816,9 @@ void Staff::write(XmlWriter& xml) const
     if (_mergeMatchingRests) {
         xml.tag("mergeMatchingRests", _mergeMatchingRests);
     }
+    if (!visible()) {
+        xml.tag("isStaffVisible", visible());
```

Comment:
Can we use this tag instead?
xml.tag("invisible", isLinesInvisible(Fraction(0, 1)));

Reply:
@RomanPudashkin, there is a difference between making a staff invisible (via the Instruments panel) and making just the staff *lines* invisible (via right-click Staff/Part Properties). Unfortunately the "invisible" tag is already being used a few lines above to make the lines invisible so it's not available to make the staff itself invisible, if that's what you were suggesting?